docs/manual-EN/1.overview/1.concepts/1.data-model.md

```diff
@@ -43,13 +43,20 @@ Edges are used to connect vertices. Each edge usually represents a relationship
 
 `Each edge` is an instance of an edge type. Our example uses _**serve**_ and _**like**_ as edge types. Take edge _**serve**_ for example, in the preceding picture, vertex `101` (represents a **player**) is the source vertex and vertex `215` (represents a **team**) is the target vertex. We see that vertex `101` has an outgoing edge while vertex `215` has an incoming edge.
 
-## Properties
+## Properties of Vertices and Edges
 
-Properties are named-value pairs within vertices and edges. In our example graph, we have used the properties `id`, `name` and `age` on **player**, `id` and `name` on **team**, and `likeness` on _**like**_ edge.
+Both vertices and edges can have properties. Properties are described with key value pairs. In our example graph, we have used the properties `id`, `name` and `age` on **player**, `id` and `name` on **team**, and `likeness` on _**like**_ edge.
 
 ## Edge Rank
 
 Edge rank is an immutable user-assigned 64-bit signed integer. It affects the edge order of the same edge type between two vertices. The edge with a higher rank value comes first. When not specified, the default rank value is zero. The current sorting basis is "binary coding order", i.e. 0, 1, 2, ... 9223372036854775807, -9223372036854775808, -9223372036854775807, ..., -1.
+In addition to an edge type, the edge between two vertices must have an edge rank. The edge rank is a 64-bit integer assigned by the user; if not specified, the edge rank defaults to 0.
+
+An edge can be represented uniquely with the [source vertex, edge type, edge rank and the dest vertex].
```

Comment:
[source vertex, edge type, edge rank, destination vertex].

[] indicates this is a list, so I think "and the" can be removed, and "destination vertex" is more friendly to readers.